### PR TITLE
Feat ref 30254 related addon adjustements

### PR DESCRIPTION
**Pull request by bakridemos**
**Ticket:** https://yaits.demos-deutschland.de/T30254

Description: adjustement to make demosplan maillane usebale as bundle:

- implement contracts
- remove the relation table 'statement_import_email_processed_attachments':  the property 'processedAttachments' has been remeoved too from the StatementImportEmail Entity, this property is not used or setted yet. this was made for The attachments of an email that are currently being processed by the PDF importer or have been finished processing. This has to be redone when this fonctionnality is needed

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Link all relevant tickets

### CODE REVIEW

**Review by AlexMenschoid — APPROVED** [review]
lgtm, code review only

**Review by dresslerdemos — APPROVED** [review]
Code quality review only.

**Comment by mussbach** [comment]
@bakridemos what happens with this PR? Should it be merged?

**Comment by bakridemos** [comment]
i think i have to check it one more time, i think i have to make some adjustements here too.

File: client/js/components/procedure/basicSettings/DpAllowedSenderEmailList.vue [hunk 1]
```diff
@@ -8,13 +8,17 @@
 </license>
 
 <template>
-  <dp-email-list
-    :init-emails="allowedEmailAddresses"
-    form-field-name="allowedSenderEmailAddresses[][fullAddress]" />
+  <div>
```
Comment (spiess-demos) [inline client/js/components/procedure/basicSettings/DpAllowedSenderEmailList.vue:11]: This whole component, along with the twig code in the block starting at projects/ewm/templates/bundles/DemosPlanProcedureBundle/DemosPlanProcedure/administration_edit.html.twig:124, should be also moved into the addon. however i guess practical reasons speak against that?

**Review by spiess-demos — DISMISSED** [review]
lgtm! (code review frontend only)

**Pull request merged**